src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/index.jsx

```diff
@@ -43,34 +47,44 @@ export default function ConfirmedAppointmentDetailsPage() {
   const featureBreadcrumbUrlUpdate = useSelector(state =>
     selectFeatureBreadcrumbUrlUpdate(state),
   );
+  const featureFetchClaimStatus = useSelector(
+    selectFeatureTravelPayViewClaimDetails,
+  );
   const isInPerson = selectIsInPerson(appointment);
   const isPast = selectIsPast(appointment);
   const isCanceled = selectIsCanceled(appointment);
   const appointmentDate = moment.parseZone(appointment?.start);
-
   const isVideo = appointment?.vaos?.isVideo;
   const isCommunityCare = appointment?.vaos?.isCommunityCare;
   const isVA = !isVideo && !isCommunityCare;
+  const fetchClaimStatus =
+    featureFetchClaimStatus && location.pathname.includes('past');
```

Comment:
This is for when the user hits refresh on the browser and we need to re-fetch the appointment.  `isPast` needs the appointment to determine if it is a past appointment. So the only way I could see to determine if we need to fetch claim data is to check the path from the the router.

It looks like avs is added to the call on any details view when hitting refresh. Not just past appointments. So there might need to be a change made there to prevent that. @JunTaoLuo 

It works for the appointments list because there's a router that routes to `PastAppointmentsList` which calls a specific method for fetching past appointments that includes the avs argument.